WordPress/Classes/ViewRelated/Menus/Views/Menu+ViewDesign.h

```diff
@@ -0,0 +1,12 @@
+
+extern CGFloat const MenusDesignStrokeWidth;
+extern CGFloat const MenusDesignDefaultCornerRadius;
+extern CGFloat const MenusDesignDefaultContentSpacing;
+
+#import "Menu.h"
+
+@interface Menu (ViewDesign)
```

Comment:
Maybe document a bit, just so the casual reader knows why we have this extension.